### ISSUE TITLE
[Material You] "Select a folder" dialog - directory info row uses wrong color

### ISSUE THREAD

**Issue by min7-i** [FossifyOrg/Gallery#208]
### Checklist

- [X] I can reproduce the bug with the latest version given [here](https://github.com/FossifyOrg/Gallery/releases/latest).
- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Gallery/issues) or [closed](https://github.com/FossifyOrg/Gallery/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Gallery/discussions) or [closed](https://github.com/FossifyOrg/Gallery/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Gallery/blob/master/CONTRIBUTING.md).

### Affected app version

1.1.3

### Affected Android/Custom ROM version

Android 14 / GrapheneOS

### Affected device model

Google Pixel 6a

### How did you install the app?

F-Droid / IzzyOnDroid

### Steps to reproduce the bug

1. Make sure you're using the Material You theme.
2. Select an image.
3. In the overflow menu, choose "copy to" or "move to".
4. Select the "Choose a folder" option.
5. Check the color of the directory info row.

### Expected behavior

The background color of the directory info row matches the background color of the rest of the dialog.

### Actual behavior

The background color of the directory info row is different than the background color of the rest of the dialog:
<img src="https://github.com/FossifyOrg/Gallery/assets/97251923/629cb8c7-1bc6-4e5f-8a66-8fa6ce81ad7f" width=50% />

**Comment by min7-i** [comment]
Sorry, I forgot to mention that the correct color was used in previous versions, e.g. in 1.1.2. That's why I considered it a bug and not an enhancement.

**Comment by connyduck** [comment]
I think this is easily fixed by removing the background [here](https://github.com/FossifyOrg/Commons/blob/88a349fe1d27b10cc2afcede53ffef2741987810/commons/src/main/kotlin/org/fossify/commons/views/Breadcrumbs.kt#L181C1-L181C65), I'm just worried that might break something else. Which tests can I do to ensure that is not the case?

**Comment by naveensingh** [comment]
@connyduck that's for the first breadcrumb's background, not the whole horizontal scroll view. Applying the proper background color on the view itself should fix it but I haven't checked what's caused this in the first place. Probably some dependency update.

**Comment by connyduck** [comment]
@naveensingh No. Maybe it should be the background of the first breadcrumb, but if you look at the code closely you will notice that `this` is `Resources` which doesn't have a `background` setter, and neither has the outer `this` which is `ItemBreadcrumbFirstBinding`. So the background is actually set to the whole `Breadcrumbs` class. 

[This commit](https://github.com/FossifyOrg/Commons/commit/ccce65da89186856d7ac03f6ee0ec365c5037fda) introduced the problem. Before the refactoring `this` actually was the view of the first breadcrumb.

Can I send a PR with a fix?

**Comment by naveensingh** [comment]
Right, I only had a quick peek. 

It should be `root.background` instead of just `background`. You are welcome to raise a PR but do test the change once with different light/dark theme variants.
